### ISSUE TITLE
Add descriptions to settings

### ISSUE THREAD

**Issue by Theblackcat98** [dessalines/thumb-key#885]
### Describe your suggested feature

As the title states, it would be nice to get better descriptions to what some of the settings do (especially since as a new user, some of the functions can be intuitive at the beginning).
More specifically what prompted me to open this issue were under **Behavior**:
- Drag-and-Return 
- Circular drag 
- Clockwise drag action 
- Counterclockwise drag action

### Acknowledgements

- [X] I have written a short but informative title.
- [X] I have updated the app to **[the latest version](https://github.com/user/repo/releases/latest)**.
- [X] I have checked through the app settings for my feature.
- [X] I have searched the existing issues and this is a new one, **NOT** a  duplicate or related to another open issue.
- [X] This is a **single** feature request, in case of multiple feature  requests I will open a separate issue for each one (they can always link to each other if related)
- [X] This is not a question or a discussion, in which case I should have  gone to [lemmy.ml/c/thumbkey](https://lemmy.ml/c/thumbkey)
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements. 🤡
- [X] I will fill out all of the requested information in this form.

**Comment by dessalines** [comment]
These already exist here: https://github.com/dessalines/thumb-key?tab=readme-ov-file#drag-and-return

**Comment by Theblackcat98** [comment]
I see the description there, yes, I still think the settings could use a better description as the functionality is either broken or I'm doing it wrong.
The "drag and return" does not work on the outside keys, I only got it to work on the inside circle and the center key.
The clockwise drag is also broken, I set it to number key for both CW and ACW and It only types letters.
Given these experiences, the description would add the confidence that the user is executing the gestures appropriately and therefore realize broken features
